client/state/comments/test/fixtures/mock-data.js

```diff
@@ -0,0 +1,10 @@
akshmii","site_ID":96330859},"date":"2016-01-26T03:24:31-08:00","URL":"http:\/\/wanderhomeblog.com\/2016\/01\/17\/the-art-of-staying-still\/comment-page-5\/#comment-740","short_URL":"http:\/\/wp.me\/p5lrs5-gk%23comment-740","content":"<p>Beautifully written Teesa love&#8230;<\/p>\n<p>Sharing my perspective after reading your blog&#8230;&#8230;<\/p>\n<p>Stillness is the only truth in this the wandering world,Everything that wanders now, has come from stillness..<\/p>\n<p>&#8220;When our identities become like iron around us, unshakable and unshedable, how do we break free?&#8221;  That&#8217;s a thought provoking question which is surrounding most of us &#8230;.<br \/>\nNext question arises as what can be done to make us free?What is possible from our side to break free?Does it depend only on me or by the people around me too?<\/p>\n","status":"approved","parent":false,"type":"comment","like_count":1,"i_like":false,"meta":{"links":{"self":"https:\/\/public-api.wordpress.com\/rest\/v1.1\/sites\/78992097\/comments\/740","help":"https:\/\/public-api.wordpress.com\/rest\/v1.1\/sites\/78992097\/comments\/740\/help","site":"https:\/\/public-api.wordpress.com\/rest\/v1.1\/sites\/78992097","post":"https:\/\/public-api.wordpress.com\/rest\/v1.1\/sites\/78992097\/posts\/1012","replies":"https:\/\/public-api.wordpress.com\/rest\/v1.1\/sites\/78992097\/comments\/740\/replies\/","likes":"https:\/\/public-api.wordpress.com\/rest\/v1.1\/sites\/78992097\/comments\/740\/likes\/"}}},{"ID":738,"post":{"ID":1012,"title":"The Art of Staying Still","type":"post","link":"https:\/\/public-api.wordpress.com\/rest\/v1.1\/sites\/78992097\/posts\/1012"},"author":{"ID":100166440,"login":"jujukarsmall","email":false,"name":"jujukarsmall","first_name":"julie","last_name":"muthui","nice_name":"jujukarsmall","URL":"http:\/\/jujukarsmall.wordpress.com","avatar_URL":"https:\/\/1.gravatar.com\/avatar\/d4ae26f0ba03495c110b18567a369d00?s=96&d=https%3A%2F%2F1.gravatar.com%2Favatar%2Fad516503a11cd5ca435acc9bb6523536%3Fs%3D96&r=G","profile_URL":"http:\/\/en.gravatar.com\/jujukarsmall","site_ID":105877261},"date":"2016-01-26T01:01:33-08:00","URL":"http:\/\/wanderhomeblog.com\/2016\/01\/17\/the-art-of-staying-still\/comment-page-5\/#comment-738","short_URL":"http:\/\/wp.me\/p5lrs5-gk%23comment-738","content":"<p>I love this cz am a fun of traveling<\/p>\n","status":"approved","parent":false,"type":"comment","like_count":0,"i_like":false,"meta":{"links":{"self":"https:\/\/public-api.wordpress.com\/rest\/v1.1\/sites\/78992097\/comments\/738","help":"https:\/\/public-api.wordpress.com\/rest\/v1.1\/sites\/78992097\/comments\/738\/help","site":"https:\/\/public-api.wordpress.com\/rest\/v1.1\/sites\/78992097","post":"https:\/\/public-api.wordpress.com\/rest\/v1.1\/sites\/78992097\/posts\/1012","replies":"https:\/\/public-api.wordpress.com\/rest\/v1.1\/sites\/78992097\/comments\/738\/replies\/","likes":"https:\/\/public-api.wordpress.com\/rest\/v1.1\/sites\/78992097\/comments\/738\/likes\/"}}}]};
```

Comment:
I don't think you need to have all this data in the responses. It's handy to just grab something from the API, but I would rather have specific, clean cases that express requirements in the tests.